.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
we shouldn't commit .vscode files. If necessary, please update the .gitignore file